### DISCUSSION TITLE
How can I join the community slack channel?

### DISCUSSION THREAD

**Discussion by jcauserjt** [dagster-io/dagster#12213] [Community]
When I go to dagster.slack.com I get the following error:

```
This link is no longer active
To join this workspace, you’ll need to ask the person who originally invited you for a new link.
```

Or sometimes it asks me for my email and I get this error message:

```
<email>@gmail.com doesn’t have an account on this workspace.
```

**Comment by jcauserjt** [comment]
Also, to clarify, my email is also an account on dagster cloud but still no luck :(

**Reply by frasermarlow** [reply]
Hi.  Sorry about that.  We ran into an authentication snag with Slack yesterday, but the issue has now been resolved.  Could you please retry and let us know if it is still giving you a hard time?  Thanks.

**Reply by jcauserjt** [reply]
Works, thank you!!

**Comment by rptechcxo** [comment]
I'm having the similar issue. Accept after it says:

`<email>@gmail.com doesn’t have an account on this workspace.`

it transfers me to an authentication page where it ask for a six digit code. When I go to my email to look for that code I met with error message instead:

 
![Screenshot 2023-05-23 at 2 09 35 PM](https://github.com/dagster-io/dagster/assets/105891310/3877d571-cdd7-4de6-a9b6-812e3d5734a7)

**Reply by oriems** [reply]
having a related issue 
<img width="735" alt="image" src="https://github.com/dagster-io/dagster/assets/6248986/2b0e3612-365c-4a7d-a9ba-7c0c8c0938a0">

**Comment by Jyoti492** [comment]
same issue with me any resolution for the same ?

**Comment by tacastillo** [comment]
Hi folks!

We try to make sure that the invite link is always valid, but sometimes they expire without us noticing.

The best way to get an up-to-date invite link to the Slack is by visiting http://dagster.io/slack

We've recently made a change that should keep this link working for the foreseeable future.
